src/admin.rs

```diff
@@ -129,12 +130,56 @@ impl Service {
                 )
                 .await),
                 "/logging" => Ok(handle_logging(req).await),
+                "/" => Ok(handle_dashboard(req).await),
                 _ => Ok(empty_response(hyper::StatusCode::NOT_FOUND)),
             }
         })
     }
 }
 
+async fn handle_dashboard(_req: Request<Body>) -> Response<Body> {
+    let apis = &[
+        (
+            "debug/pprof/profile",
+            "build profile using the pprof profiler (if supported)",
+        ),
+        (
+            "debug/gprof/profile",
+            "build profile using the gperftools profiler (if supported)",
+        ),
+        (
+            "debug/gprof/heap",
+            "collect heap profiling data (if supported)",
+        ),
+        ("quitquitquit", "shut down the server"),
+        ("config_dump", "dump the current Ztunnel configuration"),
+        ("logging", "query/changing logging levels"),
+    ];
+
+    let mut api_rows = String::new();
+
+    for (index, (path, description)) in apis.iter().enumerate() {
+        api_rows.push_str(&format!(
+            "<tr class=\"{row_class}\"><td class=\"home-data\"><a href=\"{path}\">{path}</a></td><td class=\"home-data\">{description}</td></tr>\n",
+            row_class = if index % 2 == 1 { "gray" } else { "vert-space" },
+            path = path,
+            description = description
+        ));
+    }
+
+    let html = fs::read("src/assets/dashboard.html").unwrap();
```

Comment:
We need to embed it, this file won't exist in the docker image (and shouldn't). There are rust macros to embed